src/util.coffee

```diff
@@ -32,3 +32,25 @@
                                       node?.tag is 'tag:yaml.org,2002:map'
 
 @NON_PRINTABLE = /[^\x09\x0A\x0D\x20-\x7E\x85\xA0-\uD7FF\uE000-\uFFFD]/
+
+@stringifyJSONCircularDepth = (json, depth = 1000) ->
```

Comment:
IMHO, max depth is a hack.  Circular dependency checking is simple.  

At the same time, no human generated circular dependencies will have depth > 500.  So, it really doesn't matter.